strr-host-pm-web/app/utils/host-permit-dash-helpers.ts

```diff
@@ -20,7 +20,7 @@ export const getHostPermitDashOwners = (): ConnectAccordionItem[] => {
         ),
         ...(owner.taxNumber
```

Comment:
Should `isCraNumberOptional` also be checked in condition?

Reply:
`isCraNumberOptional` is not available here because it is not stored.